common/test/acceptance/pages/studio/course_info.py

```diff
@@ -1,16 +1,169 @@
 """
 Course Updates page.
 """
-
 from common.test.acceptance.pages.studio.course_page import CoursePage
+from common.test.acceptance.pages.studio.utils import type_in_codemirror, set_input_value
 
 
 class CourseUpdatesPage(CoursePage):
     """
     Course Updates page.
     """
-
     url_path = "course_info"
 
     def is_browser_on_page(self):
-        return self.q(css='body.view-updates').present
+        """
+        Returns whether or not the browser on the page and has loaded the required content
+        """
+        return self.q(css='.handouts-content').present
```

Comment:
Can you add a comment explaining why you had to rely on handouts-content? My understanding is that you needed something that does not appear until the page is fully loaded, and there was nothing in the course updates portion of the page that you could rely on.